README.md

```diff
@@ -67,7 +67,7 @@ Basic Usage
 
 ## [Shallow Rendering](/docs/api/shallow.md)
 
-```jsx
+```javascript
```

Comment:
this will break syntax highlighting for the jsx examples. What is this intended to "fix"?